app/routes.tsx

```diff
@@ -46,6 +46,7 @@ import { getCollections } from "./components/collections/sideEffect";
 import AuthorSearch from "./containers/authorSearch";
 import { getAuthorSearchData } from "./containers/authorSearch/sideEffect";
 import { checkAuthStatus } from "./components/auth/actions";
+import axios from "axios";
```

Comment:
wrong & redundant import statement.
Remove this line and change the above line with
`import Axios, { CancelToken } from "axios"`